sdk/tests/conformance/extensions/ext-frag-depth.html

```diff
@@ -244,6 +247,62 @@
     shouldBe('gl.getExtension("EXT_frag_depth").myProperty', '2');
 }
 
+function runDeferredLinkTests() {
+    debug("");
+    debug("Testing deferred shader compilation tests.");
+
+    // Test for compilation failures that are caused by missing extensions
+    // do not succeed if extensions are enabled during linking. This would
+    // only happen for deferred shader compilations.
+
+    // First test if link succeeds with extension enabled.
+    var glEnabled = wtu.create3DContext();
+    var extEnabled = glEnabled.getExtension("EXT_frag_depth");
+
+    var vertexShader = wtu.loadShaderFromScript(glEnabled, "goodVertexShader");
+    var fragmentShader = wtu.loadShaderFromScript(glEnabled, "macroFragmentShader");
+
+    if (!vertexShader || !fragmentShader) {
```

Comment:
fragmentShader will be null here if compilation happens eagerly in the WebGL implementation and EXT_frag_depth wasn't supported.